Backpack/src/main/java/net/skyscanner/backpack/chip/BpkDismissibleChip.kt

```diff
@@ -0,0 +1,38 @@
+package net.skyscanner.backpack.chip
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.graphics.drawable.DrawableCompat
+import androidx.core.widget.TextViewCompat
+import net.skyscanner.backpack.R
+
+open class BpkDismissibleChip @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyleAttr: Int = R.style.Bpk_chip
+) : BpkChip(context, attrs, defStyleAttr) {
+
+  private var onDismissListener: OnDismissListener? = null
```

Comment:
This doesn't seem to be set anywhere? Why are we using this?

Reply:
The user will need to set this. Its a wrapper around onClickListener.

Reply:
but it's always null here, how is this supposed to be used? Unless I'm missing something this or not adding the click listener will result in the same, so we can remove all this.